src/components/tests/Leaderboard.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Leaderboard from '../Leaderboard';
+
+const leaderboard = (
+  <Leaderboard />
+);
+
+it('renders correctly', () => {
+  expect(leaderboard).toMatchSnapshot();
+});
```

Comment:
- [ ] Following the requirements, you should implement unit tests for your Redux code and integration tests for your application(checking components, mock functionality that request for data from the API).

![image](https://user-images.githubusercontent.com/79876959/147312498-46261485-20f4-432c-ac7d-73eb0dc30920.png)

Reply:
✔️